polylogue/cli/click_app.py

```
@@ -129,8 +171,20 @@ def _run_click_help(env: CommandEnv, ctx: click.Context, topic: Optional[str], e
 @click.pass_context
 def cli(ctx: click.Context, plain: bool, interactive: bool) -> None:
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)